packages/opentelemetry-plugin-http/src/utils.ts

```diff
@@ -207,4 +204,13 @@ export class Utils {
 
     return { origin, pathname, method, optionsParsed };
   }
+
+  /**
+   * Makes sure options is of type string or object
+   * @param options for the request
+   */
+  static isValidOptionsType(options: unknown): boolean {
+    const type = options && typeof options;
```

Comment:
I find name `type` is misleading here as it is may be also `false` or an empty string.

Reply:
Hi,
It can't be an empty string, it will be false (see tests https://github.com/open-telemetry/opentelemetry-js/pull/273/files#diff-b234751bed684f80529a7a50dbf4f52dR205)

Reply:
What if instead of `const type = options && typeof options;` you instead do:
```
if (!options) return false;
const type = typeof options;
return type === 'string' || (type === 'object' && !Array.isArray(options));
```
?

Reply:
sure